cpp/open3d/utility/ProgressBar.h

```diff
@@ -15,30 +18,57 @@ namespace utility {
 class ProgressBar {
 public:
     ProgressBar(size_t expected_count,
-                const std::string &progress_info,
+                std::string progress_info,
```

Comment:
Why has the const been removed here?

Reply:
It has to be copied into the object, so it passed by value into the constructor and then by move into the member variable.